app/javascript/packages/document-capture/components/documents-step.tsx

```diff
@@ -22,7 +30,13 @@ export function DocumentCaptureSubheaderOne({ selfieCaptureEnabled }) {
   );
 }
 
-export function SelfieStepWithHeader({ defaultSideProps, selfieValue }) {
+export function SelfieCaptureWithHeader({
+  defaultSideProps,
+  selfieValue,
+}: {
+  defaultSideProps: DefaultSideProps;
+  selfieValue: ImageValue;
```

Comment:
I think we try to avoid `any` as much as we can so I pushed 89b88e55317a8c4796e79740e6e3bb3ed423f3f8, it passed `npm run typecheck`

Reply:
Thanks! I wasn't figuring that out easily. Appreciate the help!